frontend/src/config/request.ts

```diff
@@ -19,19 +15,22 @@ apiRequest.interceptors.request.use((config) => {
   const mfaTempToken = getMfaTempToken();
   const token = getAuthToken();
   const providerAuthToken = SecurityClient.getProviderAuthToken();
-  
-  if (signupTempToken && config.headers) {
-    // eslint-disable-next-line no-param-reassign
-    config.headers.Authorization = `Bearer ${signupTempToken}`;
-  } else if (mfaTempToken && config.headers) {
-    // eslint-disable-next-line no-param-reassign
-    config.headers.Authorization = `Bearer ${mfaTempToken}`;
-  } else if (token && config.headers) {
-    // eslint-disable-next-line no-param-reassign
-    config.headers.Authorization = `Bearer ${token}`;
-  } else if(providerAuthToken && config.headers) {
-    // eslint-disable-next-line no-param-reassign
-    config.headers.Authorization = `Bearer ${providerAuthToken}`;
+
+  if (config.headers) {
```

Comment:
We didn't actually. I was doing some work in this file which didn't make it into the PR, but I decided to leave the code cleanup. Now it just checks that `config.headers` is present _once_, instead of once for each if statement.